csharp/test/IceSSL/configuration/AllTests.cs

```diff
@@ -147,123 +147,7 @@ public static Test.ServerFactoryPrx allTests(Test.TestHelper helper, string test
         Dictionary<string, string> d;
         try
         {
-            string[] args = new string[0];
-
-            Console.Out.Write("testing manual initialization... ");
```

Comment:
manual initialization, was about initializing the IceSSL plug-in programmatically, which is not supported since we remove the Plug-in API.